evaluate/task/task-transpile_test.go

```diff
@@ -0,0 +1,350 @@
+			task, err := TaskForIdentifier(IdentifierTranspile)
+			require.NoError(t, err)
+			tc.Task = task
+
+			tc.Validate(t,
+				func(logger *log.Logger, testDataPath string, repositoryPathRelative string) (repository evaltask.Repository, cleanup func(), err error) {
+					return TemporaryRepository(logger, testDataPath, repositoryPathRelative)
+				},
+			)
+		})
+	}
+
+	t.Run("Transpile Java into Go", func(t *testing.T) {
+		{
+			temporaryDirectoryPath := t.TempDir()
+
+			repositoryPath := filepath.Join(temporaryDirectoryPath, "golang", "transpile", "cascadingIfElse")
+			require.NoError(t, osutil.CopyTree(filepath.Join("..", "..", "testdata", "golang", "transpile", "cascadingIfElse"), repositoryPath))
+
+			modelMock := modeltesting.NewMockCapabilityTranspileNamed(t, "mocked-model")
+
+			transpiledSourceFilePath := filepath.Join("cascadingIfElse.go")
+			transpiledSourceFileContent := bytesutil.StringTrimIndentations(`
+				package cascadingIfElse
+
+			 	func cascadingIfElse(i int) int {
+			 		if i == 1 {
+			 			return 2
+			 		} else if i == 3 {
+			 			return 4
+			 		} else {
+			 			return 5
+			 		}
+			 	}
+			`)
+			modelMock.RegisterGenerateSuccess(t, transpiledSourceFilePath, transpiledSourceFileContent, metricstesting.AssessmentsWithProcessingTime).Once()
+
+			validate(t, &tasktesting.TestCaseTask{
+				Name: "Single test case",
+
+				Model:          modelMock,
+				Language:       &golang.Language{},
+				TestDataPath:   temporaryDirectoryPath,
+				RepositoryPath: filepath.Join("golang", "transpile"),
+
+				ExpectedRepositoryAssessment: map[evaltask.Identifier]metrics.Assessments{
+					IdentifierTranspile: metrics.Assessments{
+						metrics.AssessmentKeyCoverage:        30,
+						metrics.AssessmentKeyFilesExecuted:   1,
+						metrics.AssessmentKeyResponseNoError: 1,
+					},
+					IdentifierTranspileSymflowerFix: metrics.Assessments{
+						metrics.AssessmentKeyCoverage:        30,
+						metrics.AssessmentKeyFilesExecuted:   1,
+						metrics.AssessmentKeyResponseNoError: 1,
+					},
+				},
+				ExpectedResultFiles: map[string]func(t *testing.T, filePath string, data string){
+					filepath.Join(string(IdentifierTranspile), "mocked-model", "golang", "golang", "transpile.log"): func(t *testing.T, filePath, data string) {
+						assert.Contains(t, data, "PASS: TestSymflowerCascadingIfElse/#00")
+						assert.Contains(t, data, "PASS: TestSymflowerCascadingIfElse/#01")
+						assert.Contains(t, data, "PASS: TestSymflowerCascadingIfElse/#02")
+					},
+				},
+			})
+		}
+		{
+			temporaryDirectoryPath := t.TempDir()
+
+			repositoryPath := filepath.Join(temporaryDirectoryPath, "golang", "transpile")
+			require.NoError(t, osutil.CopyTree(filepath.Join("..", "..", "testdata", "golang", "transpile", "cascadingIfElse"), filepath.Join(repositoryPath, "cascadingIfElse")))
+			require.NoError(t, osutil.CopyTree(filepath.Join("..", "..", "testdata", "golang", "transpile", "sort"), filepath.Join(repositoryPath, "sort")))
```

Comment:
I think this is now renamed, right? Why is the CI not red :thinking:

Reply:
Same for the Java case.

Reply:
Only the files got renamed, that's the folder name